src/gmx2qmmm/jobs/mm.py

```diff
@@ -311,3 +307,20 @@ def read_mm_forces(self):
                         self.mmforces.append(mmforceline)
                         mmforceline = []
                 break  # read only one line
+
+    def execute_gmx(self, command_list, input_data=None):
+        # call gmx in a seperate function to be able to mock it
+        process = subprocess.Popen(command_list, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.STDOUT, universal_newlines=True)
```

Comment:
Any reason not to use `subprocess.run` (https://docs.python.org/3/library/subprocess.html#subprocess.run) here for simplicity?